### PR TITLE
Don't wrap the entrypoint doc contents in the hidden div

### PR DESCRIPTION
**Pull request by usergenic**
This should fix the worst of the symptoms of #412.  Bundling still synthesizes the output document, but the solution here was to modify the inlining code so that it handled the entrypoint doc differently when importing its contents.

I tried a different tack before this, which was to not synthesize a new document, but rather, append imports etc into the entrypoint document.  That turned out to be fraught with issues and I think this solution is far cleaner.

 - [ ] CHANGELOG.md has been updated

### CODE REVIEW

File: src/bundler.ts [hunk 1]
```diff
@@ -300,37 +310,53 @@ export class Bundler {
       reachedImports.add(resolvedUrl);
       this.rewriteImportedUrls(importDoc, resolvedUrl, docUrl);
 
-      let importParent: ASTNode;
       // TODO(usergenic): remove the remove() call when PolymerLabs/dom5#35 is
```
Comment (FredKSchott) [inline src/bundler.ts]: looks like there was an update to #35, still needed? https://github.com/Polymer/dom5/issues/35

File: src/bundler.ts [hunk 2]
```diff
@@ -300,37 +310,53 @@ export class Bundler {
...
-       * inline them.
-       *
-       * Finding them first saves on deduplication, moving them before inlining
-       * them
-       * allows the hidden div to always be found.
-       */
       const nestedImports = dom5.queryAll(importDoc, matchers.htmlImport);
-      astUtils.insertAllBefore(importParent, htmlImport, importDoc.childNodes!);
+
+      // If we're not hiding the import, because we're building the bundle
```
Comment (FredKSchott) [inline src/bundler.ts]: nit: these two comments were a little hard to follow/understand. Would love if they could be cleaned up a bit

File: src/bundler.ts [hunk 3]
```diff
@@ -548,12 +571,6 @@ export class Bundler {
         this._appendImport(hiddenDiv, newUrl);
       }
     }
-
-    /**
-     * If there's only one entrypoint, get all of its DOM into the synthetic
-     * import.
-     */
```
Comment (usergenic) [inline src/bundler.ts]: This comment was meaningless and didn't even relate to the code below it.

File: src/bundler.ts [hunk 4]
```diff
@@ -284,6 +291,9 @@ export class Bundler {
       return;
     }
 
+    const doc = dom5.nodeWalkAncestors(htmlImport, (node) => !node.parentNode)!;
+    const body = dom5.query(doc, dom5.predicates.hasTagName('body'))!;
```
Comment (usergenic) [inline src/bundler.ts]: There was some weird and fiddly code in the beforeHiddenDiv/afterHiddenDiv/inHiddenDiv logic that also found the hiddenDiv in different ways.  It is simpler to just query the doc body for it, which is why we get body here.

File: src/bundler.ts (hunk 1 above)
Comment (usergenic) [inline src/bundler.ts]: Good point.  Removed comment and removed 3 unneeded remove calls.

**Comment by usergenic** [comment]
Discovered as a result of this work is the issue #415 which I'll solve in a subsequent PR.
